src/SprykerEco/Zed/Optivo/Dependency/Facade/OptivoToLocaleFacadeBridge.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+namespace SprykerEco\Zed\Optivo\Dependency\Facade;
+
+class OptivoToLocaleFacadeBridge implements OptivoToLocaleFacadeInterface
+{
+    /**
+     * @var \Spryker\Zed\Locale\Business\LocaleFacadeInterface
+     */
+    protected $localeFacade;
+
+    /**
+     * @param \Spryker\Zed\Locale\Business\LocaleFacadeInterface $localeFacade
+     */
+    public function __construct($localeFacade)
+    {
+        $this->localeFacade = $localeFacade;
+    }
+
+    /**
+     * @return string|null
+     */
+    public function getCurrentLocaleName(): ?string
```

Comment:
Please use exactly the same method typehinting like in `LocaleFacadeInterface`.

Reply:
sorry, why I need do it?